main/webapp/modules/core/langs/translation-ar.json

```diff
@@ -195,6 +195,7 @@
     "core-index-parser/json-parser": "انقر على أول عقدة JSON { } المقابلة للسجل الأول المراد تحميله.",
     "core-index-parser/preserve-empty": "الاحتفاظ بالقيم الفارغة",
     "core-index-parser/store-archive": "حفظ ملف الأرشيف",
+    "core-index-parser/force-text": "Import all cells as text",
```

Comment:
It is not necessary to add English strings to other language files, because there is already an implicit fallback mechanism in place. In fact, by adding those English strings there, we are making it impossible for Weblate to flag those strings as needing translation - so we should definitely remove those.

Reply:
Thanks for the heads up. I'll remove them before removing the "Draft" label. I've updated the instructions in the wiki to reflect this guidance - https://github.com/OpenRefine/OpenRefine/wiki/Translate-OpenRefine